dot/rpc/helpers_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package rpc
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
+	"github.com/ChainSafe/gossamer/lib/trie"
+	"github.com/ChainSafe/gossamer/lib/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func newTestGenesisWithTrieAndHeader(t *testing.T) (
+	gen genesis.Genesis, genesisTrie trie.Trie, genesisHeader types.Header) {
+	genesisPath := utils.GetGssmrV3SubstrateGenesisRawPathTest(t)
```

Comment:
would suggest to add `t.Helper()` at the beginning of each `newTestGenesisWithTrieAndHeader` test function
```suggestion
func newTestGenesisWithTrieAndHeader(t *testing.T) (
	gen genesis.Genesis, genesisTrie trie.Trie, genesisHeader types.Header) {
	t.Helper()
	genesisPath := utils.GetGssmrV3SubstrateGenesisRawPathTest(t)
```